modules/nf-core/pbsv/tests/main.nf.test

```diff
@@ -0,0 +1,68 @@
+nextflow_process {
+
+    name "Test Process PBSV"
+    script "../main.nf"
+    process "PBSV"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pbsv"
+
+    test("pbsv_bam") {
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path +  'genomics/homo_sapiens/pacbio/bam/NA03697B2_downsampled.pbmm2.repeats.bam', checkIfExists: true)
+                ]
+
+                input[1] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome3.fasta', checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match("versions") },
+                { assert process.out.vcf != null },
```

Comment:
can you try out this [nft-vcf plugin](https://github.com/seppinho/nft-vcf) and see if you can make better assertions for vcf files with it

Reply:
- I split the pbsv module now into two modules (discover and call) - so changed this PR to be for the discover functionality specifically, where svsig.gz files are produced - and so the test no longer checks for a VCF